src/js/components/DataTable/Body.js

```diff
@@ -103,8 +103,8 @@ const Body = forwardRef(
                         }
                       : undefined
                   }
-                  onMouseOver={onClickRow ? () => setActive(index) : undefined}
-                  onMouseOut={
+                  onMouseEnter={onClickRow ? () => setActive(index) : undefined}
```

Comment:
So that's simply fewer events and fewer calls to setActive? I'm still a little surprised the redudant setActive() calls cause a re-render. Seems like a decent improvement.

Reply:
yep